code/src/components/Sidebar.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable linebreak-style */
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const Sidebar = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleSidebar = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <Container>
+      <Button onClick={toggleSidebar}>{isOpen ? '←' : 'Need tips?'}</Button>
```

Comment:
The tips here are my favorite part!! they way this is can be hide is very clean and also very useful information  ( I use Pomodoro a lot! c:)